libcore/tests/MarlinIconInfoTests/MarlinIconInfoTests.vala

```diff
@@ -0,0 +1,46 @@
+/*
+* Copyright (c) 2017 elementary LLC
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public
+* License as published by the Free Software Foundation; either
+* version 3 of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+* General Public License for more details.
+*
+* You should have received a copy of the GNU General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+* Boston, MA 02111-1307, USA.
+*
+* Authored by: Jeremy Wootten <jeremy@elementaryos.org>
+*/
+
+void add_icon_info_tests () {
+    Test.add_func ("/MarlinIconInfo/goffile_icon_update", goffile_icon_update_test);
```

Comment:
I've noticed that this is what the other tests do as well, is there a reason why this can't be called directly in main?